components/mitsubishi_uart/muart_packet-derived.cpp

```diff
@@ -202,5 +204,21 @@ std::string ThermostatHelloRequestPacket::getThermostatVersionString() const {
   return buf;
 }
 
+// ErrorStateGetResponsePacket functions
+std::string ErrorStateGetResponsePacket::getShortCode() const {
+  const auto upperAlphabet = "AbEFJLPU";
+  const auto lowerAlphabet = "0123456789ABCDEFOHJLPU";
+  const auto errorCode = this->getRawShortCode();
```

Comment:
Explicit type for this one at least (I guess the above two are okay since they're entirely contained in this method)